pywr/parameters/_activation_functions.pyx

```diff
@@ -80,7 +82,7 @@ cdef class RectifierParameter(Parameter):
         if self._value <= 0.0:
             return 0.0
         else:
-            return self.max_output * self._value / self._upper_bounds
+            return self.min_output + self._range * self._value / self._upper_bounds
```

Comment:
Is the `_range` as an attribute really needed?

` return self.min_output + (self.max_output - self.min_output) * self._value / self._upper_bounds`

Reply:
yeah, you're right. changed